src/contexts/Api/defaults.ts

```diff
@@ -9,6 +9,30 @@ import {
   APIContextInterface,
   ConnectionStatus,
 } from 'contexts/Api/types';
+import { NetworkName } from 'types';
+
+/**
+ * Checking values of "NETWORKS" instead of "NetworkName", because they're not
+ * compatible between runtime (the former) and compile time (the latter), the former
+ * having less fields defined and being the only safe one to use. This is obviously
+ * a wrong types problem, but fixing these types requires a bigger refactor.
+ */
+const isNetworkName = (value: unknown): value is NetworkName =>
+  Object.keys(NETWORKS).includes(value as string);
+
+const defaultNetworkName =
+  (process.env.NODE_ENV === 'production' && NetworkName.AlephZero) ||
```

Comment:
Should be ```isNetworkName(NetworkName.AlephZero) &&  NetworkName.AlephZero``` to check if the network is configured.

Reply:
Done :D